pom.xml

```diff
@@ -763,5 +787,35 @@ limitations under the License.
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>java11+</id>
+      <activation>
+        <jdk>[11,)</jdk>
+      </activation>
+
+      <properties>
+        <maven.compiler.release>${javaVersion}</maven.compiler.release>
+        <!-- configuration for checkstyle to use with spotless -->
+        <checkstyle.spotless.config>config/maven_checks_nocodestyle.xml</checkstyle.spotless.config>
+      </properties>
+
+      <build>
+        <!--- newer versions of plugins requires JDK 11 -->
```

Comment:
but there is no newer version use here?
maybe `<properties>` should have `<spotless-maven-plugin.version>2.36.0</spotless-maven-plugin.version>`